pages/calculator/[slug].tsx

```diff
@@ -0,0 +1,597 @@
+            />
+          </SvgIcon>
+          previous
+        </Button>
+      )}
+
+      {isPageIncludedInStepper() && <CalcStepper />}
+
+    </Container>
+  );
+}
+
+function QandAContainer({
+  page, calculatorConfig, addToResponses, setOpen,
+}: StaticCalcProps &{
+    addToResponses: SharedCalcProps['addToResponses'],
+    setOpen: SharedCalcProps['setOpen']}) {
+  return (
+    <>
+      <Box mb={4}>
+        <PortableText
+          value={page.content}
+          components={portableTextComponent}
+        />
+      </Box>
+
+      <Container id="choices-container" maxWidth="xs" sx={{ mb: 4 }}>
+        <Stack gap={2}>
+
+          {page.choices
+          && page.choices.map((choice) => {
+            const linkTo = choice.linkTo
+              ? `/calculator/${choice.linkTo.slug.current}`
+              : '#';
+            const href = choice.isExternalLink ? choice.url : linkTo;
+            return (
+              <Button
+                key={choice._key}
+                variant="contained"
+                color="primary"
+                href={href}
+                sx={{ width: '100%' }}
+                onClick={() => addToResponses(choice.label)}
+              >
+                {choice.label}
+              </Button>
+            );
+          })}
+
+          {page.isQuestion && (
+          <Button
+            variant="outlined"
+            color="primary"
+            sx={{ width: '100%' }}
+            onClick={() => setOpen(true)}
+          >
+            {calculatorConfig.notSureAnswer.promptText}
+          </Button>
+          )}
+
+        </Stack>
+      </Container>
+    </>
+  );
+}
+
+function CheckAnotherConviction({ calculatorConfig }: {
+  calculatorConfig: StaticCalcProps['calculatorConfig']
+  }) {
+  return (
+    <Box id="check-another-conviction-container">
+      <Link
+        sx={{
+          textAlign: 'center', whiteSpace: 'nowrap', display: 'flex', gap: 1,
+        }}
+        href={
+          calculatorConfig.checkAnotherConviction.linkTo.slug.current
+        }
+      >
+        <HistoryIcon />
+        {calculatorConfig.checkAnotherConviction.linkText}
+      </Link>
+    </Box>
+  );
+}
+
+function FeedbackContainer({ page, calculatorConfig }: StaticCalcProps) {
+  return (
+    <Button
+      variant="contained"
+      color="primary"
+      href={
+        page.isUndetermined
+          ? calculatorConfig.feedback.isUndeterminedUrl
+          : calculatorConfig.feedback.allOtherFeedbackUrl
+            }
+      sx={{ ml: 0, mr: 0 }}
+    >
+      {calculatorConfig.feedback.linkText}
+    </Button>
+  );
+}
+
+function ShareCalcContainer({ setShare, calcFirstPageUrl, justify }: {
+  setShare: SharedCalcProps['setShare'],
+  calcFirstPageUrl: SharedCalcProps['calcFirstPageUrl'],
+  justify?: boolean,
+}) {
+  return (
+    <Box
+      id="share-calc-container"
+      sx={{
+        display: 'flex',
+        gap: 1,
+        justifyContent: justify ? 'center' : 'flex-start',
+      }}
```

Comment:
Can get rid of all these styles

Reply:
I removed the gap but the others are needed on xs and sm.